src/augment.cpp

```diff
@@ -147,6 +147,9 @@ void augment(MutablePathMutableHandleGraph* graph,
     if (out_translations != nullptr) {
         *out_translations = make_translation(graph, node_translation, added_nodes, orig_node_sizes);
     }
+
+    // optimize the graph.  VG will use this to rebuild its in-memory path index which is important. 
```

Comment:
Is this important for correctness? Or just for speed? I'm worried the optimize method might be picking up some new semantics.